src/CMakeLists.txt

```diff
@@ -215,57 +246,34 @@ set(src_capnp_specs_rel
   nupic/proto/VectorFileSensorProto.capnp
 )
 
+# Create custom command for generating C++ code from .capnp schema files.
+
 foreach(spec ${src_capnp_specs_rel})
   list(APPEND src_capnp_specs ${PROJECT_SOURCE_DIR}/${spec})
-  list(APPEND src_capnp_hdrs ${PROJECT_BINARY_DIR}/${spec}.h)
-  list(APPEND src_capnp_srcs ${PROJECT_BINARY_DIR}/${spec}.c++)
+  list(APPEND src_capnp_generated_hdrs ${PROJECT_BINARY_DIR}/${spec}.h)
```

Comment:
Had to rename src_capnp_srcs, because it sounded like we were linking with capnproto sources when seeing the old name elsewhere in the sources, which was confusing.